table/server/underdb/hive/pom.xml

```diff
@@ -151,6 +151,10 @@
               <groupId>org.apache.logging.log4j</groupId>
               <artifactId>log4j-slf4j-impl</artifactId>
             </exclusion>
+            <exclusion>
+              <groupId>com.google.protobuf</groupId>
+              <artifactId>protobuf-java</artifactId>
+            </exclusion>
```

Comment:
is this added now to be consistent with the other dependency? Can we make this a dependency in table/server or table/server/underdb and share it between hive and glue so we don't have to name the exclusions list twice?

Reply:
my goal is to eventually remove the dependency on hive from glue. i think right now it uses some string manipulation functions from hive....